docs/testing.rst

```diff
@@ -541,10 +541,10 @@ conditions that hard to reproduce on real server::
    :type app: aiohttp.web.Application
 
    :param writer: object for managing outcoming data
-   :type writer: aiohttp.StreamWriter
+   :type writer: codecs.StreamWriter
```

Comment:
No, I'm pretty sure that it's a thing defined in aiohttp. Have you even tested the original type in REPL? What about the source?

Reply:
It's an assumption, I didn't find it in `docs.aiohttp.org` but I found it in `docs.python.org`

Reply:
You should look at the context. Most of the time, the name exists in the project but is undocumented. We shouldn't point at absolutely unrelated things from a different docs site. If it says that it's from aiohttp, it's unlikely to magically move into the CPython docs.

Reply:
Sorry for my inexperience